src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -265,28 +262,11 @@ public void autonomousExit() {
 
     @Override
     public void simulationInit() {
-        PhysicsSim physicsSim = PhysicsSim.getInstance();
-        // TODO Find more accurate values
-        if (CLIMB_ENABLED) {
-            // Motor, acceleration time from 0 to full in seconds, max velocity
-            physicsSim.addTalonFX(hardware.climbMotorFixed, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-            // physicsSim.addTalonFX(hardware.climbMotorDynamic, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-        }
-        if (INTAKE_ENABLED) {
-            physicsSim.addTalonFX(hardware.intakeMotor, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-        }
-        if (INDEX_ENABLED) {
-            physicsSim.addTalonFX(hardware.ingestIndexMotor, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-            physicsSim.addTalonFX(hardware.feederIndexMotor, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-        }
-        if (SHOOTER_ENABLED) {
-            physicsSim.addTalonFX(hardware.flywheelMotor1, 3, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-            physicsSim.addTalonFX(hardware.flywheelMotor2, 3, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-            physicsSim.addTalonFX(hardware.turretMotor, 1, 6000 * TalonFXConstants.RPM_TO_VELOCITY);
-            // Motor, stall torque, maximum free RPM
-            physicsSim.addSparkMax(hardware.hoodMotor, SparkMaxConstants.STALL_TORQUE,
-                    SparkMaxConstants.FREE_SPEED_RPM);
-        }
+        PhysicsSim sim = PhysicsSim.getInstance();
+        subsystems.climbSubsystem.simulationInit(sim);
+        subsystems.indexSubsystem.simulationInit(sim);
+        subsystems.intakeSubsystem.simulationInit(sim);
+        subsystems.shooterSubsystem.simulationInit(sim);
```

Comment:
I would rename the subsystem `simulationInit()` methods to something like `addSimulationMotors()` to make their purpose a bit clearer and avoid confusion about whether [`Subsystem`](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html) has a `simulationInit` method to parallel [`IterativeRobotBase.simulationInit()`](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/IterativeRobotBase.html#simulationInit()).

Reply:
Yeah I just named them the same as robot because I didn't know a good name

Reply:
I'll just rename them to `simInit` because it describes what they're doing without being the same as the other overrideable method